components/Users/Memberships/CancelMembership.js

```diff
@@ -53,13 +63,14 @@ mutation cancelMembership(
 export default class CancelPledge extends Component {
   constructor(props) {
     super(props)
+    const { cancellation } = props
     this.state = {
       isOpen: false,
       immediately: false,
-      suppressConfirmation: false,
-      suppressWinback: false,
-      reason: '',
-      cancellationType: ''
+      suppressConfirmation: (cancellation && cancellation.suppressConfirmation) || false,
```

Comment:
` || false` fallback not really needed since the expression before is either true or false anyway.